lib/rules/prefer-find-by.ts

```diff
@@ -464,10 +464,12 @@ export default createTestingLibraryRule<Options, MessageIds>({
               const textDestructuring = sourceCode.getText(
                 allVariableDeclarations
               );
+              const hasSpaceBeforeBracket = textDestructuring.endsWith(' }');
+              const symbolsToRemove = hasSpaceBeforeBracket ? 2 : 1;
```

Comment:
This could potentially be any number of spaces right? Would be worth it to implement this in a generic way so we calculate how many spaces there are between the last util and the closing bracket?

What do you think?

Reply:
@Belco90 yes, regex will do it best. Fixed in 41a8264